extensions/objects/templates/image-editor.component.ts

```diff
@@ -272,7 +272,6 @@ export class ImageEditorComponent implements OnInit, OnChanges {
     this.entityType = this.contextService.getEntityType();
 
     window.addEventListener('mouseup', (e) => {
-      e.preventDefault();
       this.userIsDraggingCropArea = false;
```

Comment:
?

Reply:
@heyimShivam in the overview 2nd point i have explained in detail about the change.